include/veriblock/pop/entities/endorsement.hpp

```diff
@@ -98,7 +98,7 @@ struct Endorsement {
 
   std::string toPrettyString(size_t level = 0) const {
     return format(
-        "{}{}Endorsement{id={}, containing={}, endorsed={}, blockOfProof={}}",
+        "{}{}Endorsement{{{id={}, containing={}, endorsed={}, blockOfProof={}}}}",
```

Comment:
Remove extra open and closing brace. Should be Endorsement{{

Reply:
fixed